src/CMakeLists.txt

```diff
@@ -62,8 +62,8 @@ else()
 endif()
 
 if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 13 AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS_EQUAL 13.99)
-        message(WARNING "Disabling 'reserved-identifier' warning for clang 13 (version is ${CMAKE_CXX_COMPILER_VERSION}).")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 13 AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS_EQUAL 14.99)
+        message(WARNING "Disabling 'reserved-identifier' warning for clang 13/14 (version is ${CMAKE_CXX_COMPILER_VERSION}).")
```

Comment:
Has this been necessary on your system? Which OS have you been using?

We using clang 14 in the CI runner and it wasn't necessary so far.

Reply:
I got the reserved-identifier warning on my system using:

macOS: Monterey 12.6
clang-Version: 14.0.6

By including clang version 14 in the test above, the issue got resolved.

Reply:
Interesting. And scary :D